TypeTreeDumper/Attributes/RegisterDumperPluginAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using TypeTreeDumper.Interfaces;
+
+namespace TypeTreeDumper.Attributes
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public class RegisterDumperPluginAttribute : Attribute
+    {
+        public IDumperPlugin Plugin { get; }
+
+        public RegisterDumperPluginAttribute(IDumperPlugin plugin) => Plugin = plugin;
+    }
+}
```

Comment:
This needs to use a `Type`, it can't take an instance:
```cs
[AttributeUsage(AttributeTargets.Assembly)]
public sealed class RegisterDumperPluginAttribute : Attribute
{
    public Type PluginType { get; }

    public RegisterDumperPluginAttribute(Type pluginType)
    {
        PluginType = pluginType;
    }
}
```
also nit: the attribute should go in `TypeTreeDumper`, the `Attributes` sub-namespace seems superfluous

Reply:
I'll change this